docs/install/Knative-with-OpenShift.md

```diff
@@ -5,219 +5,50 @@ weight: 15
-
-oc cluster up --server-loglevel=5
-```
-
-Once the cluster is up, login as the cluster administrator:
-
-```shell
-oc login -u system:admin
-```
-
-Now, we'll set up the default project for use with Knative.
-
-```shell
-oc project default
-
-# SCCs (Security Context Constraints) are the precursor to the PSP (Pod
-# Security Policy) mechanism in Kubernetes.
-oc adm policy add-scc-to-user privileged -z default -n default
-
-oc label namespace default istio-injection=enabled
-```
-
-## Installing Istio
-
-> Note: [Gloo](https://gloo.solo.io/) is available as an alternative to Istio.
-> [Click here](./Knative-with-Gloo.md) to install Knative with Gloo.
-
-Knative depends on Istio. First, run the following to grant the necessary
-privileges to the service accounts istio will use:
-
-```shell
-oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z default -n istio-system
-oc adm policy add-scc-to-user anyuid -z prometheus -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-egressgateway-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-citadel-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-ingressgateway-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-cleanup-old-ca-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-mixer-post-install-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-mixer-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-pilot-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z istio-sidecar-injector-service-account -n istio-system
-oc adm policy add-cluster-role-to-user cluster-admin -z istio-galley-service-account -n istio-system
-oc adm policy add-scc-to-user anyuid -z cluster-local-gateway-service-account -n istio-system
-```
-
-Run the following to install Istio:
-
-```shell
-curl -L https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml \
-  | sed 's/LoadBalancer/NodePort/' \
-  | oc apply --filename -
-```
-
-Monitor the Istio components until all of the components show a `STATUS` of
-`Running` or `Completed`:
-
-```shell
-oc get pods -n istio-system
-```
+## Before you begin
 
-It will take a few minutes for all the components to be up and running; you can
-rerun the command to see the current status.
+* An OpenShift 4 cluster is required for installation. Visit [try.openshift.com](try.openshift.com) for information on setting up a cluster. You will need cluster administrator privileges to install and use Knative on an OpenShift cluster. 
 
-> Note: Instead of rerunning the command, you can add `--watch` to the above
-> command to view the component's status updates in real time. Use CTRL+C to
-> exit watch mode.
+> Note: Currently, long-running clusters are not supported on OpenShift 4.
```

Comment:
Why's that? Does this limitation apply to OpenShift 4 or just some dev environments?